### PR TITLE
Use DHT ServerExt

### PR DESCRIPTION
**Pull request by kurnevsky**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -240,7 +238,7 @@ fn run_udp(cli_config: &CliConfig, dht_pk: PublicKey, dht_sk: &SecretKey, udp_on
     };
 
     let mut server = UdpServer::new(tx, dht_pk, dht_sk.clone());
```
Comment (NamsooCho) [inline src/main.rs]: Maybe, rename `server` to `udp_server` or `dht_server` ?

Comment (kpp) [reply src/main.rs]: right. There is already `let mut tcp_server = TcpServer::new();`

Comment (kurnevsky) [reply src/main.rs]: done

**Comment by kpp** [comment]
> If you want to change default log level you can do it via setting `RUST_LOG`
environment variable. For example, if you want to see all received and sent
packets you can change log level to `trace` for `tox_node`:
```sh
RUST_LOG=tox_node=trace cargo run --release
```

You should also point out how to change tox=info/trace .... to log packets

**Comment by kurnevsky** [comment]
Is it fine now? Or should I add more info?

**Comment by kpp** [comment]
it's ok! Ready to merge

**Pull request merged**